GliaWidgets/Sources/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -83,7 +83,7 @@ class ChatViewModel: EngagementViewModel, ViewModel {
         let uploader = FileUploader(
             maximumUploads: Self.maximumUploads,
             environment: .init(
-                uploadFile: .toEngagement(environment.uploadFileToEngagement),
+                uploadFile: { .toEngagement(environment.uploadFileToEngagement)} ,
```

Comment:
```suggestion
                uploadFile: { .toEngagement(environment.uploadFileToEngagement) },
```

Reply:
This line was deleted so it's not necessary to fix anymore.